src/Console/CommandCollection.php

```diff
@@ -34,6 +35,7 @@ class CommandCollection implements IteratorAggregate, Countable
      * Command list
      *
      * @var array
```

Comment:
```suggestion
     * @var (\Cake\Console\Shell|\Cake\Console\CommandInterface|string)[]
```
Can we at least do this?